packages/recomponents/src/components/r-tabs/id-helpers.js

```diff
@@ -0,0 +1,7 @@
+export function generateTabId(panelId) {
+    return `tab-${panelId}`;
+}
+
+export function generateTabpanelId(id) {
+    return `tabpanel-${id}`;
+}
```

Comment:
this seems like overkill- can we put these into the SFC as computed properties directly? We can always refactor them out again if a need arises.

Reply:
I believe @anthonylebrun's intention was for this to grow into an a11y helpers file and eventually move it to `recomponents/src/common/helpers` so it can be re-used in other components. 

As it stands now, especially after the naming convention refactoring - it's now way too specific to be re-use in a none tabs scenario :)